po/glossary/pt.po

```diff
@@ -434,7 +435,7 @@ msgstr "declaração financeira"
 
 #. "Free software is a matter of liberty, not price … see https://en.wikipedia.org/wiki/Free_software"
 msgid "free software"
-msgstr "programa livre"
+msgstr "Software livre"
```

Comment:
@fellen "Software" is NOT Portuguese. Using this word would kill the whole purpose of translating.
If you don't like "Programa" use "Aplicação".

Reply:
@fellen after a bit of thought and research, I decided that I will no longer maintain the translation. Main reason being the lack of respect shown for me and your own rules, as stated [here](https://wiki.gnucash.org/wiki/Translation#Contact_the_maintainer_of_your_language).
But also because I'm getting old and bitter ;)
That said, I bid you all the best with this marvelous project. Please pass on my best regards to all the team.

Reply:
> @fellen "Software" is NOT Portuguese. Using this word would kill the whole purpose of translating. If you don't like "Programa" use "Aplicação".

I agree completely with you.

But you should discuss the change with Hugo or just revert it in weblate. You are the pt_PT revisor there.

Reply:
> @fellen after a bit of thought and research, I decided that I will no longer maintain the translation. Main reason being the lack of respect shown for me and your own rules, as stated [here](https://wiki.gnucash.org/wiki/Translation#Contact_the_maintainer_of_your_language). But also because I'm getting old and bitter ;) That said, I bid you all the best with this marvelous project. Please pass on my best regards to all the team.

1. This change was about the **glossary**, not the real translation.  At some point you decided to use the translationproject and we configured it, that languages maintained there are not visible in weblate.
2. That wiki section was written before we started to use weblate, too. IIRC you have been the first, who asked, if we could offer it. 
2.1. Perhaps you can help me to _configure the access_ that it cover all desires. We are a free hosted  project there, so access restrictions are limited.
2.2 After that is finished, you help me by _updating  the wiki page_.